SETUP_ARCH.md

```diff
@@ -0,0 +1,23 @@
+# Snowdrift setup instructions for Arch Linux
+
+Arch has the benefit of being very well-documented, so this document is
+very short!
```

Comment:
Ideal documentation doesn't make documents *longer* just to express how short they are ;)

I think this is superfluous and should go away. Let shortness speak for itself.